app/tasks/report/ReportNotificationTask.scala

```diff
@@ -45,13 +45,19 @@ class ReportNotificationTask(
 
   def runPeriodicNotificationTask(taskDate: LocalDate, period: Period): Future[Unit] = {
```

Comment:
What is the use for taskDate ? I think it may not be needed anymore

Reply:
Bien vu, c'était utilisé encore pour la date affichée en haut du mail. Il n'y a pas de raison de générer la date courante deux fois, je factorise